lib/src/Uncertainty/Algorithm/Analytical/openturns/SORMResult.hxx

```diff
@@ -52,17 +52,17 @@ public:
   /** EventProbabilityBreitung accessor */
   Scalar getEventProbabilityBreitung() const;
 
-  /** EventProbabilityHohenBichler accessor */
-  Scalar getEventProbabilityHohenBichler() const;
+  /** EventProbabilityHohenbichler accessor */
+  Scalar getEventProbabilityHohenbichler() const;
```

Comment:
you must deprecate the old accessor

Reply:
How do you deprecate an accessor?

Reply:
see https://github.com/openturns/openturns/commit/dd670abb8fd8d9713016f559bfff09c67da54007

Reply:
Done.